ServerTests/ListenersTests/TcpListenerAdapterTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using FluentAssertions;
+using System;
+using System.Net;
+using Server.Clients;
+using Server.Listeners;
+using Xunit;
+
+namespace WebServerTests.ListenersTests
+{
+    public class TcpListenerAdapterTests
+    {
+        [Fact]
+        public void Ctor_Should_Create_Instance_When_ServerOptions_Valid()
```

Comment:
you test that creating TcpListenerAdapter doesnt throw exception, but dont show it in test name